velocity-tools-view/src/main/java/org/apache/velocity/tools/view/ServletUtils.java

```diff
@@ -80,9 +80,9 @@ public static String getPath(HttpServletRequest request)
         // will return the original (wrong) URI requested.  The following special
         // attribute holds the correct path.  See section 8.3 of the Servlet
         // 2.3 specification.
-        String path = (String)request.getAttribute("javax.servlet.include.servlet_path");
+        String path = (String)request.getAttribute("jakarta.servlet.include.servlet_path");
         // also take into account the PathInfo stated on SRV.4.4 Request Path Elements
```

Comment:
These comments seem to outdated. We should either make them version-neutral or remove altogether.

Reply:
done. I also noticed that a perhaps bigger issue in the future will be to refactor the security setup